addons/project/models/project_project.py

```diff
@@ -55,7 +59,7 @@ def _compute_task_count(self):
             for project, count in self.env['project.task'].with_context(
                 active_test=any(project.active for project in self)
             )._read_group(
-                [('is_closed', '=', False), ('project_id', 'in', self.ids)],
+                [('state', 'not in', list(CLOSED_STATES.keys())), ('project_id', 'in', self.ids)],
```

Comment:
```suggestion
                [('state', 'not in', list(CLOSED_STATES)), ('project_id', 'in', self.ids)],
```
This also works btw :slightly_smiling_face: